zulip_bots/zulip_bots/bots/monkeytest/doc.md

```diff
@@ -0,0 +1,8 @@
+#Monkeytest bot
+The Monkeytest bot is a bot for [Monkeytest](monkeytest.it)—service for testing sites.
+
+You can write to bot
+```
+@Monkeytest google.com
+```
+And get link to results on monkeytest.it.
```

Comment:
@Smehnov: I think you should try and follow [the giphy bot's doc.md](https://github.com/zulip/python-zulip-api/blob/master/zulip_bots/zulip_bots/bots/giphy/doc.md) as a model when documenting this. Include steps on how to set up the bot, what's the purpose of the `monkeytest.conf` file, etc. :) I feel like the documentation could be a more detailed in general! :)